pvlib/pvsystem.py

```diff
@@ -693,12 +695,12 @@ def sapm(module, poa_direct, poa_diffuse, temp_cell, airmass_absolute, aoi):
         (1 + module['Aimp']*(temp_cell - T0)) )
 
     dfout['v_oc'] = (( module['Voco'] +
-        module['#Series']*delta*np.log(Ee) + Bvoco*(temp_cell - T0) )
+        module['Cells in Series']*delta*np.log(Ee) + Bvoco*(temp_cell - T0) )
```

Comment:
Surprised to see the spaces in the column name. I thought that ``_parse_raw_sam_df`` was supposed to remove them. Spaces are a problem for people that like to use ``DataFrame.column_name`` (not me).